docs/installation/Installing_ACME_Responder.md

```diff
@@ -0,0 +1,186 @@
+Installing ACME Responder
+=========================
+
+## Overview
+
+This document describes the process to install an ACME responder on a PKI server that already has a CA subsystem.
+It assumes that the CA was [installed](Installing_CA.md) with the default instance name (i.e. pki-tomcat).
+
+**Note:** The ACME responder is currently a tech preview which means:
+* It is not intended for production.
+* It may corrupt your data.
+* There is no guarantee for correctness, security, or performance.
+* There is no guarantee for documentation or support.
+* The API, configuration, or the database may change in the future.
+* There may be no easy upgrade path to the future version. 
+
+## Installing SANToCNDefault Policy
+
+The SANToCNDefault is a certificate profile policy which generates
+a default subject DN for the certificate in case the CSR does not provide one.
+The subject DN will be generated from the first DNS name in the the SAN extension.
+
+This policy is needed by the ACME profile, but currently it is not installed by default in the CA,
+so it has to be added manually.
+
+To add the policy, edit /etc/pki/pki-tomcat/ca/registry.cfg as follows:
+
+```
+defaultPolicy.ids=...,sanToCNDefaultImpl
```

Comment:
What's the risk of adding this policy by default? I thought we had the discussion about adding the ACME profile by default, but I don't think we ever had the discussion of adding the policy. I think its generally useful to more people than just the ACME users, though unlikely that many CSRs will lack it. My 2c., but perhaps we could add this in the future.

Reply:
It depends. Do we consider the policy as something that we want to fully support or just an example since it was never required before until ACME?

If we do decide to support it and include it by default, we need to make sure that it is general enough that it can be used by other profiles. Also we will need to provide an upgrade script to add the policy into existing instances.

On the other hand, if we do always include all available policies anyway, would it make sense to remove the instance-specific registry.cfg and use a shared registry.cfg for all instances such that they are more consistent?

Reply:
If we include it by default there's a risk that people will create custom profiles using the policy, and since the ACME responder (including the profile & policy) is a tech preview, it's not considered stable yet so the custom profiles may break in future updates.

Reply:
IMO we should add the policy by default.  The component behaviour is conceptually simple and it may be useful not just for ACME.

Reply:
I think we can add it by default later in the production-ready version. For now I'd like to discourage people from creating LDAP-based profiles with this policy because we do not have a mechanism to upgrade things in LDAP yet. For example, we might want to use a better ID instead of `sanToCNDefaultImpl`.